keyboards/mode/m60s/info.json

```diff
@@ -178,6 +178,69 @@
                 {"matrix": [5, 11], "x": 11, "y": 4, "w": 1.5},
                 {"matrix": [5, 12], "x": 12.5, "y": 4}
             ]
+        },
+        "LAYOUT_60_ansi": {
```

Comment:
Should add this above the layout definitions:
```json
  "community_layouts": [
      "60_ansi",
      "60_hhkb"
  },
```

Reply:
added it.

Reply:
That didn't work because of a layout mismatch with `60_ansi`.  I am looking for a solution to the problem.

Reply:
I tried renaming it to `60_hhkb_ansi` to avoid conflicts with the `60_ansi` default layout (59 vs 61 keys).  Do I need to add a default layout named `60_hhkb_ansi` to do this the right way? Went through the docs but the right answer isn't immediately obvious to me.   Please advise and I'll get it fixed.   Thanks!